components/dashboard/src/teams/sso/OIDCClientConfigModal.tsx

```diff
@@ -55,17 +60,21 @@ export const OIDCClientConfigModal: FC<Props> = ({ clientConfig, onClose }) => {
             <ModalBody>
                 {clientConfig?.active && (
                     <Alert type="warning" className="mb-4">
```

Comment:
**suggestion**: Do you think using a message alert variant would fit better here?

```suggestion
                    <Alert type="message" className="mb-4">
```

| BEFORE | AFTER | 
|-|-|
| <img width="1512" alt="Screenshot 2023-05-16 at 23 20 53 copy" src="https://github.com/gitpod-io/gitpod/assets/120486/97d10a18-d427-4ad3-9c45-d3eecf1ec69a"> | <img width="1512" alt="Screenshot 2023-05-16 at 23 27 11" src="https://github.com/gitpod-io/gitpod/assets/120486/d618288c-b5da-4a49-8a13-31f9c6c329d7"> |

Reply:
Probably - before it was a warning because they _could_ edit and that could effect logins. Now they can't, so that message type makes sense to me.